Riot/Modules/Common/KeyboardAvoiding/KeyboardAvoider.swift

```diff
@@ -16,8 +16,9 @@
 
 import Foundation
 
+@objcMembers
 /// Avoid keyboard overlap with scroll view content
-final class KeyboardAvoider {
+final class KeyboardAvoider: NSObject {
```

Comment:
As we inherit from `NSObject` we need to call `super.init` in the initializer:
```
    init(scrollViewContainerView: UIView, scrollView: UIScrollView) {
        self.scrollViewContainerView = scrollViewContainerView
        self.scrollView = scrollView
        super.init()
    }
```

Reply:
Good catch, was forgotten.